config/config.go

```diff
@@ -322,7 +328,10 @@ func (config ServerConfig) GetDbURLRedacted() string {
 // GetDbPort - Gets the database port from the given configuration
 func (config ServerConfig) GetDbPort() int {
 	if config.DbURL != "" {
-		u, _ := url.Parse(config.DbURL)
+		u, err := url.Parse(config.DbURL)
+		if err != nil {
+			return 5432
```

Comment:
I'm thinking we should also return the zero value here (i.e. `0`), since a URL was passed that can't be handled.

Reply:
I was afraid that might be confusing, but I think you're right, that makes more sense.